boards/arduino/portenta_h7/doc/index.rst

```diff
@@ -56,7 +56,14 @@ The current Zephyr arduino_portenta_h7 board configuration supports the followin
 +-----------+------------+-------------------------------------+
 | IPM       | on-chip    | virtual mailbox based on HSEM       |
 +-----------+------------+-------------------------------------+
-
+| EXTFLASH  | on-chip    | qspi                                |
++-----------+------------+-------------------------------------+
+| SDRAM     | on-chip    | sdram                               |
++-----------+------------+-------------------------------------+
+| USB       | on-board   | usb-hs                              |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-board   | eth                                 |
++-----------+------------+-------------------------------------+
```

Comment:
I would like to add a paragraph on the different board revisions and LSE. Something along these lines (not a native speaker so feel free to adapt!):

> The high precision low speed external (LSE) clock is only fully supported on boards with hardware revision 4.10 or greater. By default the internal source is used; to enable the use of the external oscillator, manually specify the hardware revision when building the board (see :ref:`application_board_version` for information on how to build for specific revisions of the board).

Also, probably an empty line is required after the table? CI doc builds are down currently so I'm not sure :man_shrugging:

Reply:
Hi, how to check the board revision we have on hand?

Reply:
That is easy using the Arduino IDE and `mbed_portenta` core: just print out the output from `boardRevision()` (it's a uint16-packed major.minor combo).

In Zephyr, though, this functionality is not yet available. To get this information directly, you have to read the contents of the `revision` field of [the `PortentaBoardInfo` structure](https://github.com/arduino/ArduinoCore-mbed/blob/dd5abb6171063b2777620b9339510c63b915ce72/variants/PORTENTA_H7_M7/portenta_info.h#L3-L25) that is flashed at memory address `0x801F000`.